libs/designer-ui/src/lib/panel/panelheader/panelheader.tsx

```diff
@@ -226,7 +226,7 @@ export const PanelHeader = ({
       </div>
       {!noNodeOnCardLevel ? (
         <>
-          <div className={'msla-panel-card-header'} style={isRight ? {} : { paddingLeft: horizontalPadding }}>
+          <div className={'msla-panel-card-header'} style={isRight || isCollapsed ? {} : { paddingLeft: horizontalPadding }}>
```

Comment:
@rllyy97 - Will this remove any kind of padding in our portal which is needed?

Reply:
Only the padding when panel is on the left that I added last week. No impact to other padding.

Reply:
Just tested and looks fine for us @preetriti1